packages/project-disaster-trail/src/components/atoms/Ticker/index.js

```diff
@@ -0,0 +1,46 @@
+import { memo } from "react";
+import { PropTypes } from "prop-types";
+/** @jsx jsx */
+/** @jsx jsx */
```

Comment:
Can remove one of these

Reply:
Ha, that was Lint being weird. It kept adding them every time I saved the file, even if it was already there. Go home lint, you're drunk.